src/js/components/projects/detail.tsx

```diff
@@ -403,6 +404,7 @@ const ProjectDetail = (props: RouteComponentProps) => {
               users={project.github_users}
               removeUser={removeProjectUser}
             />
+            <ProjectStatusSteps project={project} tasks={tasks || []} />
```

Comment:
@duggiemitchell This is looking great; I think we're ready to add actions to the list where applicable? I think this probably means adding an optional "action" or "onclick" property on the `Step` type.

1. Create a Task: Probably we need to move the `isOpen` state of `TaskForm` out into the parent `ProjectDetail` component, so that we can pass through the `setIsOpen(true)` action (or similar) through to the step.
2. Assign a Developer... no action
3. Complete a task... no action
4. Submit this project... pass through the `openSubmitModal` action, but only if `!currentlySubmitting`. For that matter, we should probably also be passing through `readyToSubmit` directly, and use that instead as the logic for when this is active instead of simply `has_unmerged_commits`.
5. Merge pull request on GitHub... I think we probably just link to the PR? Might be easiest adding a new "link" (or similar) property to the `Step` type, then using `<ExternalLink>` with `project.pr_url`.